### ISSUE TITLE
Skála Nagyáruház

### ISSUE THREAD

**Issue by szepeviktor** [conedevelopment/cone-site#61]
(én nem értem a horizontális-vertikálist)

Fontos megérteni, hogy a honlapunk egy-egy oldalának látogatása során mi történik.

1. A tárhelyünkön lévő WordPress telepítés generál egy weboldalt, amit a böngészőnk betölt és megjelenít
2. Ez a weboldal hivatkozásokat tartalmaz például képekre, stíluslapokra (CSS), videókra - ezeket a böngészőnk a weboldal után tölti be

Tehát tárhelyen lévő WordPress és a böngészőnk által nyújtott élmény két **külön** tárgyalandó dolog.

### A WordPress {back-end}

Mit tehetünk, hogy a WordPress telepítésünk gyorsabban generálja ki az aktuálisan lekért weboldalt?

1. Olyan szolgáltatókat válasszunk, akiknek gyors az infrastruktúrája
2. Adjon a szolgáltató elegendő erőforrást, ne korlátozza
3. **PHP OPcache + persistent object cache** {ezeket nem tudom ember nyelven kifejezni, de ez a legfontosabb - 10×}
4. Tiltsunk le mindent a WordPress-ben, amit nem használunk
5. Nagyon kevés és _enterprise ready_ bővítményeket telepítsünk
6. Fejlesztessünk egyedi sablont a Cone-ékkal (vagy $99-ért https://www.wpbeaverbuilder.com/pricing/)

### Élmény a böngészőben {front-end}

1. A weboldal generáláson kívüli feladatokat bízzuk speciális szolgáltatókra
2. A weboldalról hivatkozott fájloknál legyen beállítva egyhónapos gyorsítótárazás a böngészőben
3. Az egyedileg fejlesztett sablonban csak kicsi, gyorsan betöltődő és szükséges fájlok legyenek

@adamlaki

**Comment by szepeviktor** [comment]
👆🏻 Ha a fentiek valamelyike nem megoldható,
akkor jön a szükségmegoldások Végtelen sora.

PHP OPCache-t és object cache-t nem lehet megosztani idegenek között, így a Legfontosabb nem lesz meg osztott tárhelyen 🤯 
Csak a 👉🏻 Cone Hosting-on 👈🏻 lesz 10×

**Comment by szepeviktor** [comment]
Ennek a cikknek lehetne egy folytatása: _Egy webes kattintás teljes életciklusa_.

Ebben lehetne tárgyalni, hogy mi történik lépésről lépésre.

**Comment by adamlaki** [comment]
Igen, amit leírtál és a folytatás is mind hasznos. Ügyes! Lehet érdemes lenne rámenni 2-3 cikkre a fentiek alapján. Amibe benne vagyok, viszont azt csak úgy, ha megfelelően visszahivatkozunk rád és lektorálod is kicsit.

**Comment by szepeviktor** [comment]
Azért írtam a fentieket, mert minden "WordPress sebesség" cikkben összekeverik a back-endet a front-enddel.

Pedig a back-end a tárhelyen fut,
a front-end meg a látogató böngészőjében!
Tehát a back-endhez (WP futás) csak annyi köze van a böngészőnek, hogy **vár rá**.
